neuralop/models/codano.py

```diff
@@ -0,0 +1,371 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch
+import numpy as np
+from ..layers.channel_mlp import ChannelMLP
+from ..layers.spectral_convolution import SpectralConv
+from ..layers.skip_connections import skip_connection
+from ..layers.padding import DomainPadding
+from ..layers.coda_blocks import CODABlocks
+from ..layers.resample import resample
+from ..layers.embeddings import GridEmbedding2D, GridEmbeddingND
+
+class CODANO(nn.Module):
+    """
+    Parameters
+    ---
+    output_variable_codimension : int
+        number of output channels corresponding to each variable
+    hidden_variable_codimension : int
+        number of hidden channels corresponding to each variable
+    lifting_variable_codimension : int
+        number of hidden channels in lifting/projection module for each variable
+    use_positional_encoding : bool
+        whether to use varibale specific positional encoding
+    positional_encoding_dim : int
+        dimension (number of channels) of positional encoding
+    static_channel_dim : int
+        number of channels in static channel. These channels contains additional static information of the PDE such as boundary conditions.
+    positional_encoding_modes : list
+        number of Fourier modes used in positional encoding
+    n_variables : int
```

Comment:
more clarity on what this means would be useful

Reply:
Added further clarifications.

Reply:
I think it could even be clearer - what are the variables? How is positional encoding performed?

Reply:
And is `n_variables` analogous to `in_channels`? If so this should be a higher-ordered arg, and its meaning to the model should be explained.

e.g., `number of distinct physical variables in data, encoded as channels`